Protos/api.proto

```diff
@@ -3,7 +3,7 @@ syntax = "proto3";
 package api;
 
 service SchedulerApi {
-    rpc SubmitTask(TaskRequest) returns (TaskResponse) {}
+    rpc SubmitTask(TaskRequest) returns (SubmitTaskToSchedulerResponse) {}
```

Comment:
> SubmitTaskToSchedulerResponse

When do we submit task to scheduler?

Reply:
That's the flow we have. Driver submits task to scheduler and get a future back. 
After this PR, We will be getting multiple futures back.

Reply:
oh ok, I mistook it for scheduler->worker